README.md

```diff
@@ -164,5 +164,69 @@ workflows:
               only: /^v.*/
 ```
 
+### push-to-app-collection
+
+This job generate an App CR and add it to the an app collection chart repository.
+
+* The App name and catalog are passed by parameters (respectively `app_name` and `app_catalog`).
+* The App version is automatically detected by `architect project version`.
+* The app collection repository where the App CR is added to is passed by parameter (`app_collection_repo`).
+
+**NOTE**: The job requires `CATALOGBOT_SSH_KEY_PRIVATE_BASE64` environment
+variable to be set in the build. This must be base64 encoded private SSH key of
+[CatalogBot Github user][catalogbot-user].
+
+**NOTE**: app collection repositories configured in the job parameters must be
+added to the [Catalog Editors][catalog-editors-team] GitHub team with write permission. See the
+paragraph below for explanation.
+
+This job assumes that the app collection is defined in a GitHub repository inside
+giantswarm organization. E.g. when `app_collection_repo` parameter is set to
+`"aws-app-collection"` the job will try to use https://github.com/giantswarm/aws-app-collection.
+All interactions with the app collection GitHub repository are done with [CatalogBot github
+user] credentials.
+
+[catalog-editors-team]: https://github.com/orgs/giantswarm/teams/catalog-editors/repositories
+[CatalogBot github user]: https://github.com/catalogbot
+
+Example usage
+
+```yaml
+version: 2.1
+orbs:
+  architect: giantswarm/architect@VERSION
+
+workflows:
+  my-workflow:
+    jobs:
+      - architect/push-to-app-collection:
+          name: "deploy via app collection"
```

Comment:
```suggestion
          name: "push-to-aws-app-collection"
```

The reason for that suggesstion is we reference those jobs are requirements for other jobs.